packages/runtime-dom/src/components/TransitionGroup.ts

```diff
@@ -97,8 +96,6 @@ const TransitionGroupImpl = {
     })
 
     return () => {
-      const rawProps = toRaw(props)
-      const cssTransitionProps = resolveTransitionProps(rawProps)
       const tag = rawProps.tag || Fragment
```

Comment:
I moved this to top, because it's static if the re-render not happen.It can improve performance.